packages/gestalt/src/ScrollableContainer.jsdom.test.js

```diff
@@ -0,0 +1,41 @@
+// @flow strict
+import React, { createRef } from 'react';
+import { render } from '@testing-library/react';
+import ScrollableContainer from './ScrollableContainer.js';
+import { useScrollableContainer } from './contexts/ScrollableContainer.js';
+import Box from './Box.js';
+
+describe('ScrollableContainer', () => {
+  it('renders correctly', () => {
+    const ref = createRef();
+
+    const TestBox = () => {
+      const { addRef, scrollableContainerRef } = useScrollableContainer();
+      return addRef && scrollableContainerRef ? <Box ref={ref} /> : null;
+    };
+
+    render(
+      <ScrollableContainer>
+        <TestBox />
+      </ScrollableContainer>,
+    );
+    expect(ref.current instanceof HTMLDivElement).toEqual(true);
```

Comment:
This does test that the ref points at a div, but it doesn't actually test that it's the div we specified. Perhaps give that Box an id and look for it here?

Reply:
kinda, merged into single test.

one is making sure children renders, the other one checks ref from context was received.

Reply:
ADDRESSED IN LAST COMMIT